Packs/SafeNet_Trusted_Access/Integrations/SafeNetTrustedAccess/SafeNetTrustedAccess.py

```diff
@@ -0,0 +1,935 @@
+    header_sequence = ["id", "schemaVersionNumber", "userName", "firstName", "lastName", "email", "mobileNumber",
+                       "alias1", "alias2", "custom1", "custom2", "custom3", "address", "city", "state", "country",
+                       "postalCode", "isSynchronized"]
+
+    return CommandResults(
+        readable_output=tableToMarkdown("STA user successfully created :", response, headers=header_sequence,
+                                        headerTransform=pascalToSpace, removeNull=True),
+        outputs_prefix='STA.USER',
+        outputs_key_field=['id'],
+        outputs=response
+    )
+
+
+def update_user_sta_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """ Function for sta-update-user-info command. Update profile of a specific user. """
+
+    response = client.update_user_sta(args=args)
+    if not response:
+        return CommandResults(
+            readable_output=NO_RESULT_MSG,
+        )
+    header_sequence = ["id", "schemaVersionNumber", "userName", "firstName", "lastName", "email", "mobileNumber",
+                       "alias1", "alias2", "custom1", "custom2", "custom3", "address", "city", "state", "country",
+                       "postalCode", "isSynchronized"]
+
+    return CommandResults(
+        readable_output=tableToMarkdown("STA user successfully updated:", response,
+                                        headers=header_sequence, headerTransform=pascalToSpace, removeNull=True),
+        outputs_prefix='STA.USER',
+        outputs_key_field=['id'],
+        outputs=response
+    )
+
+
+def delete_user_sta_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """ Function for sta-delete-user command. Delete user from the tenant. """
+
+    response = client.delete_user_sta(userName=args.get('userName'))
+    if not response:
+        return CommandResults(
+            readable_output=NO_RESULT_MSG,
+        )
+    return CommandResults(
+        readable_output=f"## STA user - {args.get('userName')} successfully deleted.",
+        outputs_prefix='STA.USER',
+        outputs_key_field=['id'],
+        outputs=response
+    )
+
+
+def get_user_groups_sta_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """ Function for sta-get-user-groups command. Get all the groups associated with a specific user. """
+
+    response = client.get_user_groups_sta(userName=args.get('userName'), limit=args.get('limit'))
+    if not response:
+        return CommandResults(
+            readable_output=NO_RESULT_MSG,
+        )
+    header_sequence = ['id', 'schemaVersionNumber', 'name', 'description', 'isSynchronized']
+    return CommandResults(
+        readable_output=tableToMarkdown(
+            f"Groups associated with user - {args.get('userName')} : ", response, headers=header_sequence,
+            headerTransform=pascalToSpace, removeNull=True),
+        outputs_prefix='STA.GROUP',
+        outputs_key_field=['id'],
+        outputs=response
```

Comment:
If this command return all group the user is associated to, so where we can see it in the context? You can list all users under a group, or list all user's groups under USER

Reply:
Done.
As suggested, updated the context data for **user-groups** and **group-members** commands.
![image](https://user-images.githubusercontent.com/86044314/132178794-9d89591d-7645-4967-8597-de76929ab9d7.png)
![image](https://user-images.githubusercontent.com/86044314/132178892-43f7565f-02d7-467e-aabe-7bc8ee40e933.png)